docs/Getting Started/configuration.md

```diff
@@ -12,40 +12,32 @@ There are two ways of configuring Codyze, through command line options or a conf
 
 If both are present, the command line options take precedence over the configuration file. 
 For list and map type options, the data from the configuration file can be overwritten if the normal option (e.g. `--option-name`) is used. 
-To append the data from the command line to the one from the configuration file, use the `+` option in __Codyze v2__ (e.g. `--option-name+`) or the `additions` option in __Codyze v3__ (e.g. `--option-name-additions`)
+To append the data from the command line to the one from the configuration file, use the `additions` option (e.g. `--option-name-additions`)
 
 
 ## Command Line Interface
 There are three execution modes in which Codyze can run:
+
 * Command line mode:
   <br />Non-interactive command line client, accepts arguments from command line and runs analysis
 * Language server protocol mode:
   <br />This mode is for IDE support and binds to stdout as a server for Language Server Protocol (LSP)
 * Interactive console mode:
   <br />The text based user interface (TUI) is an interactive console that allows exploring the analyzed source code by manual queries
 
-The execution mode is specified in __Codyze v2__ through an option flag (`-c`, `-l` or `-i`).
-For __Codyze v3__ subcommands are used (`analyze`, `lsp`, or `interactive`).
+Subcommands are used to enter the modes (`analyze`, `lsp`, or `interactive`).
```

Comment:
Are we still supporting all subcommands?

I remember that we decided to support only `analyze` fot now. @maximilian-galanis can you confirm?

Reply:
I think we should change that in #671 where the subcommands are removed.